tensorboard/webapp/persistent_settings/_data_source/persistent_settings_data_source.ts

```diff
@@ -84,6 +84,16 @@ export class OSSSettingsConverter extends SettingsConverter<
       serializableSettings.timeSeriesCardMinWidth =
         settings.timeSeriesCardMinWidth;
     }
+    if (settings.stepSelectorEnabled !== undefined) {
```

Comment:
Is it possible to add tests for the new logic in this file?

Reply:
The other ones are not really tested. But, I added test that hit all the new logic paths.